package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@duffel/components",
-  "version": "3.7.6",
+  "version": "3.7.7--canary-3",
```

change to 

```suggestion
  "version": "3.7.7",
```

before merging

---

done